rbc/remotejit.py

```diff
@@ -710,6 +849,22 @@ def python(self, statement):
         response = self.client(remotejit=dict(python=(statement,)))
         return response['remotejit']['python']
 
+    def normalize_function_type(self, ftype: Type):
+        """Apply RemoteJIT specific hooks to normalized function Type.
+
+        Parameters
+        ----------
+        ftype: Type
+          typesystem type of a function
+
+        Returns
+        -------
+        ftype: Type
+          typesystem type of a function with normalization hools applied
+        """
+        assert ftype.is_function, ftype
```

Comment:
Not a fan of using `assert` for control flow but I guess none of our users would call Python with `-O`.

Reply:
It is not a control flow. I am using `assert` only for quick internal sanity checks. If somebody hits it, we have two options: either there is an internal bug that we'll need to fix, or the `assert` needs to be replaced with a proper exception raise as the user input can cause the hit.

Reply:
ok I thought it was used for that because the input parameter is then just going through.

Reply:
RemoteOmnisci redefines the method to handle UDTFs that have different signatures for the implementation functions and the corresponding SQL operators.

Reply:
Ah yes I see.